### ISSUE TITLE
Upgrade Vert.x SQL client to 5

### ISSUE THREAD

**Issue by DavideD** [hibernate/hibernate-reactive#2036]

**Comment by gsmet** [comment]
Let's make sure this is synchronized with the integration of Vert.x 5 in Quarkus.

**Comment by DavideD** [comment]
Sure.
I just need to start to work on it to have an idea of the changes required in Hibernate Reactive.

**Comment by tsegismont** [comment]
> Let's make sure this is synchronized with the integration of Vert.x 5 in Quarkus.

Agreed. But keep in mind Hibernate Reactive 3 will have to be integrated in Quarkus before Vert.x 5.

So, as @DavideD suggested offlist, we could have Vert.x 5 in Hibernate Reactive 3.1
